airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -151,85 +153,150 @@ def print_job_logs(
+            aws_secret_access_key=credentials.secret_key,
+        )
+        glue_client = self.get_conn()
         next_log_token = None
+        mode = None
         job_failed = False
+        if verbose and continuous_logging:
+            mode = 'CONTINUE'
+            self.log.info("Continuous logging mode is enable.")
+        elif verbose:
+            mode = 'BATCH'
+            self.log.info("Batch logging mode is enable.")
+        else:
+            self.log.info("Logging is disable.")
 
         while True:
             try:
-                job_run_state = self.get_job_state(job_name, run_id)
-                if job_run_state in finished_states:
-                    self.log.info('Exiting Job %s Run State: %s', run_id, job_run_state)
-                    return {'JobRunState': job_run_state, 'JobRunId': run_id}
-                if job_run_state in failed_states:
-                    job_failed = True
-                    job_error_message = f'Exiting Job {run_id} Run State: {job_run_state}'
-                    self.log.info(job_error_message)
-                    raise AirflowException(job_error_message)
-                else:
-                    self.log.info(
-                        'Polling for AWS Glue Job %s current run state with status %s',
-                        job_name,
-                        job_run_state,
-                    )
-                    time.sleep(self.JOB_POLL_INTERVAL)
-            finally:
-                if verbose:
-                    next_log_token = self.print_job_logs(
+                job_run = glue_client.get_job_run(JobName=job_name, RunId=run_id, PredecessorsIncluded=True)
+                job_run_state = job_run['JobRun']['JobRunState']
+                msg = f'[Batch] The Job {job_name} with run id {run_id} is : {job_run_state}'
+                if mode == 'CONTINUE':
+                    log_group_name = job_run['JobRun']['LogGroupName']
+                    token = self.print_output_logs(
                         job_name=job_name,
                         run_id=run_id,
-                        job_failed=job_failed,
+                        log_group_name=log_group_name,
+                        log_client=log_client,
                         next_token=next_log_token,
                     )
+                    while token is not None:
+                        next_log_token = token
+                        token = self.print_output_logs(
+                            job_name=job_name,
+                            run_id=run_id,
+                            log_group_name=log_group_name,
+                            log_client=log_client,
+                            next_token=next_log_token,
+                        )
+                elif mode == 'BATCH':
+                    msg = f"The Job {job_name} with run id {run_id} is : {job_run_state}"
+                    self.log.info(msg)
+                else:
+                    self.log.info(msg)
```

Comment:
I know it's silly, but can you humour me and add a blank line here between the `if` blocks to visually separate them?
```suggestion
                    self.log.info(msg)
                    
```

Reply:
Sure will do